### ISSUE TITLE
Disable ExtraTranslation lint warning on release builds.

### ISSUE THREAD

**Issue by krt16s** [wish7code/openbmap#139]
Building the last version makes lint complain about extra translation, I fix this on my side by running `sed -i -e '/MissingTranslation/adisable "ExtraTranslation"\n' build.gradle`, but such tweaks should be handled by upstream -- or the translation stuff should be fixed itself.

The error message:

````
:app:lintVitalRelease/home/krt/src/fdroiddata/build/org.openbmap/android/app/src/main/res/values-fr/scanModeArray.xml:3: Error: "scan_mode_entries" is translated here but not found in default locale [ExtraTranslation]
    <string-array name="scan_mode_entries">
                  ~~~~~~~~~~~~~~~~~~~~~~~~
/home/krt/src/fdroiddata/build/org.openbmap/android/app/src/main/res/values-fr/scanModeArray.xml:8: Error: "scan_mode_values" is translated here but not found in default locale [ExtraTranslation]
    <string-array name="scan_mode_values">
                  ~~~~~~~~~~~~~~~~~~~~~~~

   Explanation for issues of type "ExtraTranslation":
   If a string appears in a specific language translation file, but there is
   no corresponding string in the default locale, then this string is probably
   unused. (It's technically possible that your application is only intended
   to run in a specific locale, but it's still a good idea to provide a
   fallback.).

   Note that these strings can lead to crashes if the string is looked up on
   any locale not providing a translation, so it's important to clean them
   up.

2 errors, 0 warnings
 FAILED
````

**Comment by wish7code** [comment]
Hey Boris!

I just fixed the issue with 4b0e71f0f7
Thanks for all your time and efforts @ F-Droid :-)

Cheers
Toby

**Comment by krt16s** [comment]
Thanks, 0.8.15 builds without the workaround.
